external/radiation/radiation/wrapper_api.py

```diff
@@ -182,3 +188,57 @@ def _rad_compute(self, state: State, time: cftime.DatetimeJulian,) -> Diagnostic
         )
         out = postprocess_out(out)
         return unstack(out, coords)
+
+    def _get_model(
+        self,
+        time: cftime.DatetimeJulian,
+        nz: int,
+        tracer_name_mapping: Mapping[str, str] = TRACER_NAMES_IN_MAPPING,
+    ) -> MutableMapping[Hashable, Any]:
+        model: MutableMapping[Hashable, Any] = {
+            "me": self._comm.rank,
+            "levs": nz,
+            "levr": nz,
+            "nfxr": self._rad_config.nfxr,
+            "ncld": self._rad_config.ncld,
+            "ncnd": self._rad_config.ncnd,
+            "fhswr": self._rad_config.fhswr,
+            "fhlwr": self._rad_config.fhlwr,
+            # todo: why does solar hour need to be one timestep behind time to validate?
+            "solhr": _solar_hour(time - timedelta(seconds=self._timestep)),
```

Comment:
this is the other issue you were referring to?

Reply:
Yes --  I'll make an issue in fv3net to refer to here.